packages/mdc-card/mdc-card.scss

```diff
@@ -15,174 +15,151 @@
-    height: 80px;
-    margin: 16px 0 0;
-    padding: 0;
-
-    @each $mult, $name in (1.5: "1dot5", 2: "2", 3: "3") {
-      // postcss-bem-linter: ignore
-      &--#{$name}x {
-        width: auto;
-        height: $mult * 80px;
-      }
-    }
-  }
+.mdc-card__action-buttons,
+.mdc-card__action-icons {
+  display: flex;
+  align-items: center;
+  box-sizing: border-box;
+}
 
-  &__title {
-    @include mdc-typography(body2);
-    @include mdc-theme-prop(color, text-primary-on-light);
+.mdc-card__action-icons {
+  flex-grow: 1;
+  justify-content: flex-end;
+}
 
-    // Adjust text color for dark theme.
-    @include mdc-theme-dark(".mdc-card") {
-      @include mdc-theme-prop(color, text-primary-on-dark);
-    }
+.mdc-card__action-buttons + .mdc-card__action-icons {
+  @include mdc-rtl-reflexive-box(margin, left, 16px);
+}
 
-    margin: -.063rem 0; // -1sp 0
-  }
+//
+// Action items
+//
 
-  &__title--large {
-    @include mdc-typography(headline);
+.mdc-card__action {
+  display: inline-flex;
+  align-items: center;
+  box-sizing: border-box;
+  min-width: auto;
+  margin: 0;
+  cursor: pointer;
+}
 
-    margin: 0;
-  }
+.mdc-card__action:not(:last-child) {
+  @include mdc-rtl-reflexive-box(margin, right, 8px);
+}
 
-  &__subtitle {
-    @include mdc-typography(body1);
-    @include mdc-theme-prop(color, text-primary-on-light);
+//
+// Action buttons
+//
 
-    // Adjust text color for dark theme.
-    @include mdc-theme-dark(".mdc-card") {
-      @include mdc-theme-prop(color, text-primary-on-dark);
-    }
+// stylelint-disable-next-line plugin/selector-bem-pattern
+.mdc-card__action.mdc-button {
+  padding: 0 8px;
+}
 
-    margin: -.063rem 0; // -1sp 0
+.mdc-card__actions--full-bleed .mdc-card__action {
+  display: flex;
+  justify-content: space-between;
+  width: 100%;
+  height: auto;
+  max-height: none;
+  padding: 8px 16px;
+  text-align: left;
+
+  @include mdc-rtl {
+    text-align: right;
   }
+}
 
-  &__horizontal-block {
-    @include mdc-rtl-reflexive-property(padding, 0, 16px);
-
-    display: flex;
-    flex-direction: row;
-    align-items: flex-start;
-    justify-content: space-between;
-    box-sizing: border-box;
+//
+// Action icons
+//
 
-    // postcss-bem-linter: ignore
-    .mdc-card__actions--vertical {
-      margin: 16px;
-    }
+.mdc-card__action--icon:not(:disabled) {
+  @include mdc-theme-prop(color, text-icon-on-background);
+  @include mdc-states;
+}
 
-    .mdc-card__media-item {
-      @include mdc-rtl-reflexive-box(margin, left, 16px);
-    }
+// stylelint-disable-next-line plugin/selector-bem-pattern
+.mdc-card__action--icon.mdc-button {
+  padding: 0 4px;
+}
 
-    // postcss-bem-linter: ignore
-    .mdc-card__media-item--3x {
-      margin-bottom: 16px;
-    }
-  }
+// Icon toggles are taller than buttons, so we need to adjust their margins to prevent the action row from expanding.
+// stylelint-disable-next-line plugin/selector-bem-pattern
+.mdc-card__action--icon.mdc-icon-toggle {
```

Comment:
Would it be preferable to use 2 classes both internal to this package if we need to override specificity, rather than needing to reference a class in another package? (I realize the alternative also breaks BEM though...) cc @lynnjepsen

Reply:
In this particular case, I think that might be a tad overkill.

Clients already need to add `mdc-card__action` and `mdc-card__action--icon`,
so a third class (which is only necessary if the icon is _also_ an `mdc-icon-toggle`)
would get kinda verbose:

```html
<i class="material-icons mdc-card__action mdc-card__action--icon mdc-card__action--icon-toggle" tabindex="0" role="button" title="Share">share</i>
```

That said, if you or Lynn feel strongly that we should have another modifier class instead, I can certainly add one.